infrastructure/helm/keycloak/login-setup-2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+RELEASENAME=$1
+CLIENTSECRET=$2
+LOCALIP=$(ifconfig | grep 'inet ' | sed '/127/d' | cut -f 2 -d ' ')
+KEYCLOAKPORT=$(kubectl get svc $RELEASENAME-keycloak -o=jsonpath='{.spec.ports[0].nodePort}')
+
+# add 4180 port to $RELEASENAME-kuberay-head-svc
+kubectl patch svc $RELEASENAME-kuberay-head-svc  -p '{"spec":{"ports": [{"port": 4180,"name":"proxy","appProtocol":"tcp","Protocol":"TCP","targetport": 4180}]}}'
+
+# update $RELEASENAME-kuberay
+sed "s/<CLIENT_SECRET>/$CLIENTSECRET/g" quantumserverless-kuberay-patch.yaml | sed "s/<NODEPORT>/$KEYCLOAKPORT/g" | sed "s/<LOCALIP>/$LOCALIP/g" | sed "s/<RELEASE>/$RELEASENAME/g" > patch.yaml
```

Comment:
Can we assume that the user can configure these values from its k8s cluster configuration before running the release? Can we avoid patching somehow?

Reply:
Yes, we can.